### PR TITLE
Create SECURITY.md for security reports

### LINKED ISSUE #198: Add contributor guidance for security concerns
### What is the version of your ORAS CLI

0.16.0

### What would you like to be added?

Guidance for contributors opening PRs and issues about security concerns. It would be best if people did not create public PRs and issues for security concerns.

### Why is this needed for ORAS?

To give developers time to resolve security issues before the public is informed.

### Are you willing to submit PRs to contribute to this feature?

- [X] Yes, I am willing to implement it.

### PR DESCRIPTION
**Pull request by deepeshaburse**
This PR will redirect the reader to the ORAS' security policy in the documentation.

Link to PR for security policy: #244 

It fixes #198

### CODE REVIEW

**Comment by deepeshaburse** [comment]
cc: @TerryHowe

The mail ID and number of days required for action haven't been added. Please let me know what should be written there.

File: SECURITY.md
Comment (TerryHowe) [inline SECURITY.md:7]: How do we want to report? Slack might be easiest?

**Comment by TerryHowe** [comment]
Maybe we want to put this text in the document and create a SECURITY.md in each repo that links to the web page?

File: SECURITY.md
Comment (deepeshaburse) [reply SECURITY.md:7]: Yes it will be but it won't be discreet and will defeat the purpose of asking them to separately report. Or, we'll have to give one specific person's slack to dm.

File: SECURITY.md
Comment (deepeshaburse) [reply SECURITY.md:18]: Yeah sure that sounds fine 👍🏼

**Comment by deepeshaburse** [comment]
> Maybe we want to put this text in the document and create a SECURITY.md in each repo that links to the web page?

Okay, so add it to the documentation and link it in every repo?

File: SECURITY.md
Comment (TerryHowe) [reply SECURITY.md:7]: Slack DM I think. Give some time for people to discuss I think

**Comment by deepeshaburse** [comment]
I think this would be helpful: https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability

**Comment by deepeshaburse** [comment]
The link to the webpage will not work unless PR #244 does not get merged.

**Review by TerryHowe — DISMISSED** [review]
/lgtm

Need to hold this until the other PR merges as you say.

**Review by asmitbm — APPROVED** [review]
LGTM

**Comment by deepeshaburse** [comment]
@TerryHowe and @asmitbm updated content with semantic line breaks.

**Review by sajayantony — APPROVED** [review]

File: SECURITY.md
Comment (TerryHowe) [inline SECURITY.md:10]:
```suggestion
you may follow the steps we have mentioned in ORAS's [security policy](https://oras.land/docs/community/reporting_security_concerns). 
```

**Review by TerryHowe — CHANGES_REQUESTED** [review]

**Comment by TerryHowe** [comment]
Partial https://github.com/oras-project/community/issues/11

**Review by sajayantony — APPROVED** [review]

**Pull request merged**
